parser/rule_parser.go

```diff
@@ -400,7 +388,46 @@ func (r *RuleParser) getConditionForProvider(langProvider, capability string, va
 				ignorable = i
 			}
 		}
+	}
+
+	if capability == "dependency" {
```

Comment:
Note to self's I think we may eventually want to refactor this method it has a ton of logic in it

Reply:
Sorry not this one, but the one above, maybe I am just wrong too :)